cc_library/README.md

```diff
@@ -9,4 +9,26 @@ To make sure all dependencies are loaded, run `pip install -r cc_library/require
 - To check the codestyle, run `flake8 cc_library`.
 
 ### Raspberry Pi Set up
-For setting up a Pi, [a read the `README.md`](src/sciler/README.md) in the sciler package. 
\ No newline at end of file
+For setting up a Pi, [a read the `README.md`](src/sciler/README.md) in the sciler package. 
+
+
+## Set-up Device
+- run `pip install -i https://test.pypi.org/simple/ scclib`
```

Comment:
watch the space

Reply:
from `https://test.pypi.org/simple/` install `scclib` is correct